.github/workflows/tests.yml

```diff
@@ -27,82 +27,57 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        java: [11, 17, 20, 21-ea]
-        os: [ubuntu-latest, windows-latest]
-        exclude:
-          - os: windows-latest
-            java: 17
-          - os: windows-latest
-            java: 20
-          - os: windows-latest
-            java: 21-ea
-
-
-
+        java: [11, 17, 20]
+        os: [ubuntu-latest]
```

Comment:
So we are losing windows tests?

Reply:
Never! I added a dedicated, plain-actions workflow